src/sh/keptn/Keptn.groovy

```diff
@@ -522,6 +522,23 @@ def addCustomLabels(requestBody, labels) {
     return requestBody
 }
 
+/**
+ * takes the request JSON body, adds eventpayload and reports back that JSON as string
+ */
+def addEventTypePayload(requestBody, eventType, eventPayload) {
```

Comment:
Also here, should we call it `eventTypePayload` instead of `eventPayload` for consistency?